features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -86,9 +86,6 @@
         <AuthorizationGrantCacheTimeout>-1</AuthorizationGrantCacheTimeout>
         <SessionDataCacheTimeout>-1</SessionDataCacheTimeout>
         <ClaimCacheTimeout>-1</ClaimCacheTimeout>
-		<RequestTokenUrl>https://localhost:9443/oauth/request-token</RequestTokenUrl>
```

Comment:
If we can use these configurations to define a custom URL here better to comment those lines and keep it in the file